src/walletinitinterface.h

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) 2017 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#ifndef WALLETINITINTERFACE_H
+#define WALLETINITINTERFACE_H
+
+#include <string>
+
+class CScheduler;
+class CRPCTable;
+
+class WalletInitInterface {
+public:
+    /** Get wallet help string */
+    virtual std::string GetWalletHelpString(bool showDebug) {return (std::string)"";}
```

Comment:
style-nit: Instead of casting a literal, you could directly return an empty `std::string{};`

Reply:
better. Thanks!